.github/workflows/lint.yaml

```diff
@@ -0,0 +1,44 @@
+name: Lint
+on:
+  push:
+    tags:
+      - v*
+    branches: [ main, develop ]
+  pull_request:
+permissions:
+  contents: read
+  # Optional: allow read access to pull request. Use with `only-new-issues` option.
+  # pull-requests: read
+jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.17
```

Comment:
use go version `1.19`

Reply:
only go version 1.17 is supported from the official golangci-lint workflow:
https://github.com/golangci/golangci-lint-action